Roadmap.md

```diff
@@ -1,78 +1,183 @@
-_This is a living document. It describes the priorities as they exist today and will evolve over time._
+_This is a living document containing ChakraCore team's current priorities as well as release notes for previous releases. Last updated 5/23/17._ 
 
-All the changes done in the public repository flow into Chakra and Microsoft Edge on a regular basis as described in the [Contribution guidelines](https://github.com/Microsoft/ChakraCore/blob/master/CONTRIBUTING.md). The following is a summary of the ChakraCore team's backlog for the next 6 months. Last updated 1/12/17.
+All the changes done in the public repository flow into Chakra and Microsoft Edge on a regular basis as described in the [Contribution guidelines](https://github.com/Microsoft/ChakraCore/blob/master/CONTRIBUTING.md).
 
-# Enhancing Host & Platform Support 
+# Future
+
+The following is a summary of the ChakraCore team's backlog for the next 6 months. Some completed items (either in a release or master branch) are included to provide the context and progress of the work.
+
+## Enhancing Host & Platform Support 
 * **Node.js**
   * [x] Update [Node.js+ChakraCore](https://github.com/nodejs/node-chakracore) to include [ChakraCore 1.4](https://github.com/Microsoft/ChakraCore/tree/release/1.4)
-  * [x] Implement V8 debug protocol in the Node.js [ChakraShim](https://github.com/nodejs/node-chakracore/tree/chakracore-master/deps/chakrashim) to enable debugging using VS Code.
+  * [X] Enable Node-ChakraCore on Mac and Linux.
   * [ ] Support Chrome Debug Protocol in Node-ChakraCore.
+  * [ ] Enable Time Travel Debugging over Chrome Debug Protocol using VSCode
+  * [ ] Support N-API
+  * [ ] Improve Node-ChakraCore performance based on benchmarks and real-world scenarios
```

Comment:
The following will be done in this week release 
* Support Chrome Debug Protocol in Node-ChakraCore.
* Enable Time Travel Debugging over Chrome Debug Protocol using VSCode
* Support N-API